### DISCUSSION TITLE
How to change the format of the post title?

### DISCUSSION THREAD

**Discussion by HabDV** [Rongronggg9/RSS-to-Telegram-Bot#80] [Q&A] [answered]
Greetings. How to change HTML markup for title ? Valid on `<b>`

**Comment by Rongronggg9** [comment]
IDK what do you mean exactly. Would you please describe your question in detail?

**Comment by HabDV** [comment]
rrs publishes underlined titles, is it possible to change them to hot ones 

[picture example](https://imageup.ru/img207/thumb/2022-05-20-1542463940733.jpg)

**Comment by Rongronggg9** [comment, marked as answer]
https://github.com/Rongronggg9/RSS-to-Telegram-Bot/blob/e0eaf3a35be8d8dc11174d7c19d1c0c309f50b9a/src/parsing/post_formatter.py#L397

https://github.com/Rongronggg9/RSS-to-Telegram-Bot/blob/e0eaf3a35be8d8dc11174d7c19d1c0c309f50b9a/src/parsing/post_formatter.py#L433-L436

**Reply by HabDV** [reply]
Removed (Underline and did not help

**Reply by Rongronggg9** [reply]
How did you deploy?

**Reply by HabDV** [reply]
git clone https://github.com/Rongronggg9/RSS-to-Telegram-Bot.git
cp docker-compose.yml.sample docker-compose.yml
docker-compose up -d

**Reply by Rongronggg9** [reply]
In this way, you are always using the upstream Docker image. Any local modification will NOT affect the [remote Docker image](https://hub.docker.com/r/rongronggg9/rss-to-telegram). You need to either build/publish your own Docker image or [dirty run from source](https://github.com/Rongronggg9/RSS-to-Telegram-Bot/blob/dev/docs/deployment-guide.md#option-42-dirty-run-from-source).

https://github.com/Rongronggg9/RSS-to-Telegram-Bot/blob/28b50cb23b7696194aa043573a8cbb7f433655d3/docker-compose.yml.sample#L6
